lib/bootsnap/cli.rb

```diff
@@ -144,7 +189,17 @@ def parser
         help = <<~EOS
           Path pattern to not precompile. e.g. --exclude 'aws-sdk|google-api'
```

Comment:
Maybe change the help to use multiple patterns? OTOH this shows it supports regexps…